xonsh/built_ins.py

```diff
@@ -714,6 +715,7 @@ def load_builtins(execer=None, config=None, login=False, ctx=None):
     builtins.__xonsh_ensure_list_of_strs__ = ensure_list_of_strs
     builtins.__xonsh_list_of_strs_or_callables__ = list_of_strs_or_callables
     builtins.__xonsh_completers__ = xonsh.completers.init.default_completers()
+    builtins.__xonsh_events__ = events
```

Comment:
Shouldn't this be public if we expect people to use it?  That is, `builtins.events = events`